scripts/summit-cli.mjs

```
@@ -0,0 +1,861 @@
+  const type = args._[1];
+  const name = args._[2];
+
+  if (!type || !name) {
+    console.log('Usage: summit new <service|package|component|migration> <name>');
+    return 1;
+  }
+
+  switch (type) {
+    case 'service':
+      const port = args.flags.port || 4050;
+      const owner = args.flags.owner || 'team';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)